lib/routers/establishment/projects.js

```diff
@@ -229,11 +229,24 @@ router.param('projectId', (req, res, next, projectId) => {
           constrainRopParams: builder => builder.select('id', 'year', 'status', 'submittedDate')
         });
     })
-    .then(project => {
+    .then(async project => {
       if (!project) {
         throw new NotFoundError();
       }
       project.collaborators = project.collaborators.filter(c => c.establishments.map(e => e.id).includes(req.establishment.id));
+
+      const establishmentForUser = establishment => {
+        return req.user.can('establishment.read', {establishment: establishment.id})
+          .then(canReedEstablishment => {
```

Comment:
Any reason it's canReed? To distinguish between past / present tense?

Reply:
canReed was just a type, have updated to canRead

Reply:
I thought I may have been overthinking it haha